src/engine/terrain.cpp

```diff
@@ -174,7 +184,11 @@ void Terrain::GenerateModel(Chunk* chunk, const PLVector2& offset) {
     }
   }
 
-  chunk_mesh->texture = atlas_->GetTexture();
+  //if(atlas_ == nullptr) {
```

Comment:
Ditto to the above, just needs to be `chunk_mesh->texture = atlas_->GetTexture()` again.